components/home/TestimonialsSection.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from 'react';
+import { CaretLeft, CaretRight } from 'phosphor-react';
+import Link from 'next/link';
+
+const TestimonialsSection = ({ textContent }) => {
+  const data = textContent.cards;
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const beforeIndex =
+    currentIndex !== 0 ? textContent.cards.slice(0, currentIndex) : textContent.cards.slice(4, data.length);
+  const current =
+    // currentIndex === 5
```

Comment:
Comments without explanations should be removed if they don't help to understand how the code works